package.json

```diff
@@ -14,11 +14,15 @@
   },
   "dependencies": {
     "autoprefixer-loader": "^3.1.0",
-    "babel-core": "^5.8.34",
-    "babel-loader": "^5.4.0",
+    "babel-core": "^6.3.15",
+    "babel-loader": "^6.2.0",
     "babel-plugin-react-display-name": "^2.0.0",
-    "babel-plugin-react-transform": "^1.1.1",
-    "babel-runtime": "^5.8.34",
+    "babel-plugin-react-transform": "^2.0.0-beta1",
+    "babel-plugin-transform-runtime": "^6.3.13",
+    "babel-preset-es2015": "^6.3.13",
+    "babel-preset-react": "^6.3.13",
+    "babel-preset-stage-0": "^6.3.13",
+    "babel-runtime": "^6.3.19",
```

Comment:
updated...appears that all the babel 6 stuff is written in es6 and needs the babel 5 runtime to support platforms that don't support that...i believe. here's the best reference I could find : https://phabricator.babeljs.io/T6763#67858